CordovaLib/Classes/Private/Plugins/CDVWebViewEngine/CDVWebViewEngine.m

```diff
@@ -74,9 +74,25 @@ - (WKWebViewConfiguration*) createConfigurationFromSettings:(NSDictionary*)setti
     }
 
     configuration.allowsInlineMediaPlayback = [settings cordovaBoolSettingForKey:@"AllowInlineMediaPlayback" defaultValue:NO];
-    configuration.mediaPlaybackRequiresUserAction = [settings cordovaBoolSettingForKey:@"MediaPlaybackRequiresUserAction" defaultValue:YES];
+
+    // Check for usage of the older preference key, alert, and use.
+    BOOL mediaTypesRequiringUserActionForPlayback = [settings cordovaBoolSettingForKey:@"MediaTypesRequiringUserActionForPlayback" defaultValue:YES];
+    NSString *mediaPlaybackRequiresUserActionKey = [settings cordovaSettingForKey:@"MediaPlaybackRequiresUserAction"];
+    if(mediaPlaybackRequiresUserActionKey != nil) {
+        mediaTypesRequiringUserActionForPlayback = [settings cordovaBoolSettingForKey:@"MediaPlaybackRequiresUserAction" defaultValue:YES];
+    }
+    configuration.mediaTypesRequiringUserActionForPlayback = mediaTypesRequiringUserActionForPlayback;
+
```

Comment:
These codes are for `mediaTypeRequiringUserActionForPlayback` which (already) included in other PR https://github.com/apache/cordova-ios/pull/784. 
Is this necessary for `allowsAirPlayForMediaPlayback` ?